mockfaster/src/main/java/com/decathlon/tzatziki/utils/MockFaster.java

```diff
@@ -50,6 +50,7 @@ public class MockFaster {
             .withReasonPhrase("Not Found Again!")
             .withStatusCode(404);
 
+    public static final String INVALID_REGEX_PATTERN = "(\\[\\])";
```

Comment:
```suggestion
    public static final String INVALID_REGEX_PATTERN = "(\\[])";
```
Turns out that it is not needed to escape closing bracket since it auto detects that there is no opening bracket after my testing